fitbenchmarking/core/tests/test_fitting_benchmarking_minimizers.py

```diff
@@ -133,8 +133,7 @@ def test_run_minimzers_selected(self, loop_over_hessians):
         Tests that some minimizers are selected
         """
         self.options.algorithm_type = ["general"]
-        self.results = [[self.result_args]]
```

Comment:
I don't think we need result_args to be a dict anymore? Can you change it to just creating the fitresults object in the setup?